test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -33,7 +33,7 @@ public static class E2ETest
 
             // show multiple errors with line info
             "publish-url-conflict", "output-path-conflict", "uid-conflict", "redirection-conflict",
-            "redirected-id-conflict", "circular-reference", "moniker-overlapping", "empty-monikers",
+            "redirected-id-conflict", "circular-reference", "moniker-overlapping", "empty-monikers", "moniker-range-invalid"
```

Comment:
`moniker-range-invalid` should have line info

Reply:
Add `TODO`

---
In reply to: [286339043](https://github.com/dotnet/docfx/pull/4600#discussion_r286339043) [](ancestors = 286339043)